cpp/include/cudf/unary.hpp

```diff
@@ -31,6 +33,65 @@ namespace cudf {
  * @brief Column APIs for unary ops
  */
 
+/**
+ * @brief Convert a floating-point value to fixed-point
```

Comment:
Suggestion: only hyphenate "fixed point" when it is used as a compound adjective.

```suggestion
 * @brief Convert a floating-point value to fixed point.
```